### ISSUE TITLE
Not working on python 2.7.10

### ISSUE THREAD

**Issue by pathcl** [EMC-Underground/vsphere-python-restapi#1]
Hello,

Currently I'm facing this issue.

Caught vmodl fault : [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)

I fixed inserting https://github.com/quickjp2/vsphere-python-restapi/blob/master/methods.py#L28 to https://github.com/quickjp2/vsphere-python-restapi/blob/master/methods.py#L24

**Comment by quickjp2** [comment]
Once your code is merged into the dev branch, I'll verify the change. If it's good then I'll merge your code and close the issue.

**Comment by pathcl** [comment]
@quickjp2 you can contact me at lsanmartinr@gmail.com
